amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/bulk/BulkBaseService.java

```diff
@@ -194,9 +203,49 @@ protected ResultSet getTotalItems(Query searchQuery, int skipCount)
         searchMapper.fromQuery(searchParams, searchQuery);
         searchParams.setSkipCount(skipCount);
         searchParams.setMaxItems(1);
+        searchParams.setLimit(1);
         return searchService.query(searchParams);
     }
 
+    protected final class BulkProgress
```

Comment:
I suggest extracting this to a class (or record?) of its own.

Reply:
Extracted to a record.